src/main/java/de/tum/in/www1/artemis/service/connectors/athena/AthenaSubmissionSendingService.java

```diff
@@ -110,30 +95,30 @@ public void sendSubmissions(TextExercise exercise, int maxRetries) {
     /**
      * Calls the remote Athena service to submit a Job for calculating automatic feedback
      *
-     * @param exercise        the exercise the automatic assessments should be calculated for
-     * @param textSubmissions the submissions to send
-     * @param maxRetries      number of retries before the request will be canceled
+     * @param exercise    the exercise the automatic assessments should be calculated for
+     * @param submissions the submissions to send
+     * @param maxRetries  number of retries before the request will be canceled
      */
-    public void sendSubmissions(TextExercise exercise, Set<TextSubmission> textSubmissions, int maxRetries) {
-        Set<TextSubmission> filteredTextSubmissions = new HashSet<>(textSubmissions);
+    public void sendSubmissions(Exercise exercise, Set<Submission> submissions, int maxRetries) {
+        Set<Submission> filteredSubmissions = new HashSet<>(submissions);
 
         // filter submissions with an open participation (because of individual due dates)
-        filteredTextSubmissions.removeIf(submission -> {
+        filteredSubmissions.removeIf(submission -> {
             var individualDueDate = submission.getParticipation().getIndividualDueDate();
             return individualDueDate != null && individualDueDate.isAfter(ZonedDateTime.now());
         });
 
-        if (filteredTextSubmissions.isEmpty()) {
-            log.info("No text submissions found to send (total: {}, filtered: 0)", textSubmissions.size());
+        if (filteredSubmissions.isEmpty()) {
+            log.info("No submissions found to send (total: {}, filtered: 0)", submissions.size());
             return;
         }
 
-        log.info("Calling Athena to calculate automatic feedback for {} submissions.", textSubmissions.size());
+        log.info("Calling Athena to calculate automatic feedback for {} submissions.", filteredSubmissions.size());
```

Comment:
Maybe here you can log the filtered count as well, i.e. `submissions.size() - filteredSubmissions.size()`